app/containers/TripDetailPage/selectors.js

```diff
@@ -35,12 +35,19 @@ const selectOfferId = () => createSelector(
 const selectTripForOfferDetail = () => createSelector(
   selectTripDetailPageDomain,
   (offerPageState) => {
-    console.log(offerPageState.toJS())
-
+   
     return offerPageState.toJS().tripData.trip;
   }
   );
 
+const selectLocationDataForOfferDetail = () => createSelector(
+  selectTripDetailPageDomain,
+  (offerPageState) => {
```

Comment:
You can simplify this code as:
`(offerPageState) =>  offerPageState.toJS().tripData.locationData`